publisher/server_only/README.md

```diff
@@ -55,8 +55,8 @@ The following table outlines and annotates the steps you may take to test and ex
-|  2   | Enter the user email address that you want to use for testing and click **Log In**.                                           | This is a call to the `/login` endpoint ([server.js](server.js)). The login initiated on the server side then calls the [POST /token/generate](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/endpoints/post-token-generate.md#decrypted-json-response-format) endpoint and processes the received response.                                                                                                                                                                                                                                                                                                                                                                           |
-|      | The main page is updated to display links to the two pages with protected content and the established UID2 identity information. | The displayed identity information is the `body` property of the [JSON response payload](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/endpoints/post-token-generate.md#decrypted-json-response-format) from the successful `POST /token/generate` response. If the response is successful, the returned identity is saved to a session cookie (a real-world application would use a different way to store session data) and the protected index page is rendered.                                                                                                                                                                                                                   |
+|  2   | Enter the user email address that you want to use for testing and click **Log In**.                                           | This is a call to the `/login` endpoint ([server.js](server.js)). The login initiated on the server side then calls the [POST /token/generate](https://unifiedid.com/docs/endpoints/post-token-generate#decrypted-json-response-format) endpoint and processes the received response.                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Not sure why you've linked to the "Decrypted JSON Response Format" section here. Did you mean to? If not, just remove the fragment and link to https://unifiedid.com/docs/endpoints/post-token-generate. If you intended to, no worries.